src/internal/serviceenv/service_env.go

```diff
@@ -199,19 +207,10 @@ func (env *NonblockingServiceEnv) initKubeClient() error {
 
 func (env *NonblockingServiceEnv) initDBClient() error {
 	return backoff.Retry(func() error {
-		host, ok := os.LookupEnv("POSTGRES_SERVICE_HOST")
-		if !ok {
-			return errors.Errorf("postgres service host not found")
-		}
-		portStr, ok := os.LookupEnv("POSTGRES_SERVICE_PORT")
-		if !ok {
-			return errors.Errorf("postgres service port not found")
-		}
-		port, err := strconv.Atoi(portStr)
-		if err != nil {
-			return err
-		}
-		db, err := dbutil.NewDB(dbutil.WithHostPort(host, port))
+		db, err := dbutil.NewDB(
+			dbutil.WithHostPort(env.config.PostgresServiceHost, env.config.PostgresServicePort),
+			dbutil.WithDBName(env.config.PostgresDBName),
+		)
```

Comment:
Using the config values rather than pulling from the env because why would we pull from the env.

Reply:
Lol 👍 . Where is the magic that populates the struct with all the `env` tags?

Reply:
I had to ask msteffen to point me to it - it's in `cmdutil/env.go`